scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -1305,7 +1305,7 @@ def test_slerp():
 
 
 def test_slerp_single_rot():
-    with pytest.raises(ValueError, match="must be a sequence of rotations"):
+    with pytest.raises(ValueError, match="must be a sequence of at least 2 rotations"):
```

Comment:
This is a tight long
```suggestion
    msg = "must be a sequence of at least 2 rotations"
    with pytest.raises(ValueError, match=msg):
```

Also could you add a test for the other part of the condition? We have 2 validations, hence should have 2 independent tests.

Reply:
Will do.